doc/ROS2_PX4_Offboard_Tutorial.md

```diff
@@ -0,0 +1,415 @@
+
+## Overview
+
+This tutorial explains how to interface ROS2 with PX4 (SITL) using DDS.
+
+### Prerequisites
+
+   * [ROS2 Installed](https://docs.px4.io/main/en/ros/ros2_comm.html#install-ros-2), and setup for your operating system (e.g. [Linux Ubuntu](https://docs.px4.io/main/en/dev_setup/dev_env_linux_ubuntu.html)) with Gazebo
+   * [FastDDS Installed](https://docs.px4.io/main/en/dev_setup/fast-dds-installation.html)
+   * [PX4-Autopilot downloaded](https://docs.px4.io/main/en/dev_setup/building_px4.html)
+   * [QGroundControl installed](https://docs.qgroundcontrol.com/master/en/getting_started/download_and_install.html)
+
+### Example System Details
+
+  * Ubuntu 22.02
+  * ROS2 Humble
+  * Python 3.10
+
+## Install PX4 Offboard and dependencies (one time setup)
+
+### Install the px4-offboard example from Jaeyoung-Lim
+
+```
+cd ~
+git clone https://github.com/Jaeyoung-Lim/px4-offboard.git
+```
+
+### Install PX4 ROS Com and PX4 msg
+
+The `px4-offboard` example requires the `px4_ros_com` bridge and `px4_msgs` definitions:
+
+```
+cd ~
+mkdir px4_ros_com_ws
+cd px4_ros_com_ws
+git clone https://github.com/PX4/px4_ros_com.git
```

Comment:
We don't need `px4_ros_com` for this package to work

Reply:
Removed